src/transport_performance/utils/raster.py

```diff
@@ -13,6 +13,10 @@
 
 from rioxarray.merge import merge_arrays
 from rasterio.warp import Resampling
+from transport_performance.utils.defence import (
+    _check_parent_dir_exists,
+    _is_expected_filetype,
+)
 
 
 def merge_raster_files(
```

Comment:
On a successful run, this function prints the inputs and output. This could be considered obnoxious. Is this something we want to remain? Most likely this won't be changed, especially if it is part of the `rioxarray` package.  
![image](https://github.com/datasciencecampus/transport-network-performance/assets/121952297/d718ade0-1015-4ec2-95bc-e8fabc0d7d15)

Reply:
This printing is coming from the fact that you're using a Jupyter notebook and by default the cell prints out the returned value of the function if it's not assigned to a variable. `merge_raster_files()` returns the boundaries of all the inputs and the output so that a user could check their extents should they wish e.g., to make sure there are no gaps or overlaps in the merging.

Are you happy to resolve this conversation based on this?